src/index.js

```diff
@@ -115,8 +120,7 @@ class MulticastDNS extends EventEmitter {
           data: peerInfo.id.toB58String() + '.' + serviceTag
         })
 
-        // Only announce TCP multiaddrs for now
-        const multiaddrs = tcp.filter(peerInfo.multiaddrs)
+        // TODO announce all the TCP multiaddrs
         const port = multiaddrs[0].toString().split('/')[4]
```

Comment:
Should we check the existance of `multiaddrs[0]` as well as the existance of the 4th element in the split array?